spec/support/controller_macros.rb

```diff
@@ -0,0 +1,14 @@
+module ControllerMacros
```

Comment:
I think we don't need this helper as we are using devise token auth to handle the login